src/main/java/com/currencycloud/client/CurrencyCloud.java

```diff
@@ -104,7 +104,7 @@ Account updateAccount(
     ) throws ResponseException;
 
     /** Find Account */
-    @GET
+    @POST
```

Comment:
@falconetpt this will not work since you have not changed the @QueryParam to @FormParam. I would suggest, rather than updating the existing methods, I would deprecate them, then clone them and rename to findAccountsPost etc.

Then you can change the query params to form params. Finally, you will need to update any calling code to use the new Post methods.

Of course you will need to add new test cases for these (cloning the existing should be good enough)